fleshed out test automation working group (#45)
NHSDigital/software-engineering-quality-framework: communities/pd-test-automation-working-group.md

## Scope

Areas of interest are specifically:

* Automated Functional Testing
* Automated Non-Functional Testing, e.g. load/capacity/performance/soak testing, security testing, etc
* Automated Code Hygiene. e.g. static code analysis, dead code detection, code profiling, technical debt, etc

## Coordinator

* This will be a rotating post on a 3-month basis, requiring a commitment of one day per week
* The coordinator will run a blog to help publicise the activity of the group
* There will be a deputy coordinator (to cover sessions when the coordinator is away, etc). They will also typically take over as coordinator for the following 3 months. This gives the group continuity.

> * [dlavender4](https://github.com/dlavender4) will be coordinator initially

Reviewer: For consistency, could change to:
```suggestion
* David Lavender [dlavender4](https://github.com/dlavender4) will be coordinator initially
```

* A Deputy will be found during the first sessions

## Members

* Formed from within PD
* Core members should be kept to under 15, to promote interactive sessions

> * Should include a mix of backgrounds: Dev, DevOps, Test, Assurance, etc.

Reviewer: Evil etc
```suggestion
* Should include a mix of backgrounds including Dev, DevOps, Test and Assurance.
```

> * Should include people from as many teams as possible

Reviewer: ```suggestion
* Should include people from a representative spread of teams
```

* Can join as representatives from a specific team, or as "interested parties"

## Format

* The group's official home is the "Test Automation CoP" Teams channel
* This includes a Backlog and Wiki

> * Any member can contribute to the backlog. This could be show-and-tells of tools or approaches; queries about how to do something; curation of a Principal; talking about an organisation-wide policy; etc.

Reviewer: Principle, not principal
```suggestion
* Any member can contribute to the backlog. This could be show-and-tells of tools or approaches; queries about how to do something; curation of a principle; talking about an organisation-wide policy; etc.
```

## Scope

Areas of interest are specifically:

* Automated Functional Testing
* Automated Non-Functional Testing, e.g. load/capacity/performance/soak testing, security testing, etc
* Automated Code Hygiene. e.g. static code analysis, dead code detection, code profiling, technical debt, etc

## Coordinator

* This will be a rotating post on a 3-month basis, requiring a commitment of one day per week
* The coordinator will run a blog to help publicise the activity of the group
* There will be a deputy coordinator (to cover sessions when the coordinator is away, etc). They will also typically take over as coordinator for the following 3 months. This gives the group continuity.
* [dlavender4](https://github.com/dlavender4) will be coordinator initially
* A Deputy will be found during the first sessions

## Members

* Formed from within PD
* Core members should be kept to under 15, to promote interactive sessions
* Should include a mix of backgrounds: Dev, DevOps, Test, Assurance, etc.
* Should include people from as many teams as possible
* Can join as representatives from a specific team, or as "interested parties"

> ## Time Booking

Reviewer 2: Can we drop this section please, and mention it in the first session instead? Just because this is public, it's a bit internal-NHSD-esque

Reviewer 2: When I say "this section" I just mean the time booking ABR reference :)
